tsfresh/utilities/dataframe_functions.py

```diff
@@ -11,6 +11,8 @@
 import numpy as np
 import pandas as pd
 
+import itertools
```

Comment:
why do you need this import? Did I miss something?

Reply:
Not needed, will remove